pkg/nacler/nacler_test.go

```diff
@@ -0,0 +1,58 @@
+package nacler
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"testing"
+
+	"github.com/kolide/krypto/pkg/nacler/keyers/localecdsa"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNacler(t *testing.T) {
```

Comment:
You could add the failure case -- Mallory can't unseal

Reply:
I'll add this in the next PR!